docs/operating-scylla/nodetool-commands/cluster/repair.rst

```diff
@@ -0,0 +1,82 @@
+Nodetool cluster repair
+=======================
+
+**cluster repair** - A process that runs in the background and synchronizes the data between nodes. It only repairs keyspaces with tablets enabled (default).
+To repair keyspaces with tablets disabled (vnodes-based), see :doc:`nodetool repair </operating-scylla/nodetool-commands/repair/>`.
+
+Running ``cluster repair`` on a **single node** synchronizes all data on all nodes in the cluster.
+
+To synchronize all data in clusters that have both tablets- and vnodes-based keyspaces, run :doc:`nodetool repair -pr </operating-scylla/nodetool-commands/repair/>` on **all**
+of the nodes in the cluster, and :doc:`nodetool cluster repair </operating-scylla/nodetool-commands/cluster/repair/>` on  **any** of the nodes in the cluster.
+
+To check if a keyspace enables tablets, use:
+
+.. code-block:: cql
+
+  DESCRIBE KEYSPACE `keyspace_name`
+
+.. note:: Run the :doc:`nodetool cluster repair </operating-scylla/nodetool-commands/cluster/repair/>` command regularly. If you delete data frequently, it should be more often than the value of ``gc_grace_seconds`` (by default: 10 days), for example, every week.
+
+ScyllaDB node will ensure the exclusivity of tablet repair and maintenance operations (add/remove/decommission/replace/rebuild).
+
+ScyllaDB nodetool cluster repair command supports the following options:
+
+
+- ``-dc`` ``--in-dc`` syncs data between all nodes in a list of Data Centers (DCs).
+
+
+  .. warning:: This command leaves part of the data subset (all remaining DCs) out of sync.
```

Comment:
Let's add warnings for all of the selections options. When tombstone gc repair mode is used, partial repair which does not synchronize  data between all replicas in the cluster will NOT advance the repair time for tombstone gc. It is not recommended to use the selections options to perform partial repair if it is really needed.